drivers/wifi-client/driver.js

```diff
@@ -67,6 +68,7 @@ class UnifiDriver extends Homey.Driver {
         // One poller for initializing the connection every 60 seconds (if needed)
         this.pollIntervals.push(setInterval(() => {
             this.initializeConnection();
+            this.getUsageData();
```

Comment:
Why do you fetch usage data here? Shouldn't it be part of the updateClientList and updating the tx/rx values be part of the sendDeviceUpdates method (e.g. be part of drivers/wifi-client/device.js and be updated there)?